packages/playground/worker-thread-node/main.ts

```diff
@@ -3,8 +3,8 @@ import InlineWorker from './worker-inline?worker&inline'
 import { Worker } from 'worker_threads'
 
 export const run = async (message: string): Promise<string> => {
-  return new Promise((resolve, reject) => {
-    const worker = new MyWorker() as Worker
+  return new Promise((resolve, _) => {
+    const worker = new MyWorker() as unknown as Worker
```

Comment:
Do we need to do this? It worked before :thinking: didn't it?

Reply:
Yes, we need to do this else TS throws this error.

![image](https://user-images.githubusercontent.com/4309729/130051447-9f02ad72-0efa-4b64-bacf-f463f2547166.png)

earlier it was working cause we export type as the union of web worker or node worker, now its type is web worker